src/kem/sike/kem_sike.c

```diff
@@ -70,3 +70,159 @@ OQS_KEM *OQS_KEM_sike_p751_new() {
 }
 
 #endif
+
+// Preventing redefinition of symbols
+// Todo: remove
+#undef OQS_SIDH_MSR_CRYPTO_SECRETKEYBYTES
```

Comment:
You've marked this `Todo: remove` -- is it to be removed before merging or something we should do later (which we are likely to forget)?